Readme.md

```diff
@@ -2,31 +2,29 @@
 
 [![CI](https://github.com/waldoapp/wdio-service/actions/workflows/build.yaml/badge.svg)](https://github.com/waldoapp/wdio-service/actions/workflows/build.yaml)
 
-WebdriverIO service that provides a better integration into [Waldo](https://www.waldo.com/scripting).
+A WebdriverIO service that provides better integration with [Waldo](https://www.waldo.com/scripting).
 
-What this package provide:
+This package provides:
 
-- Automatic configuration of Waldo as the webdriver endpoint
-- A set of additional commands to ease mobile testing
-- TypeScript definition for the options and mobile specific capabilities
+- Automatic configuration of Waldo as the webdriver endpoint.
+- A collectin of additional commands to ease mobile testing.
+- A TypeScript definition for the options and mobile-specific capabilities.
 
 ## Installation
 
-The easiest way is to keep `@waldoapp/wdio-service` as a devDependency in your `package.json` via:
+The easiest way to install this package is to define `@waldoapp/wdio-service` as a device dependency (`devDependency`) in your `package.json`:
```

Comment:
Please keep devDependency without description it's the key in the json object

Reply:
As you wish